### ISSUE TITLE
Variational inference for mixtures, topics, and sequences

### ISSUE THREAD

**Issue by michaelchughes** [tufts-ml/ml-research-reading-lists#4]
## Basics: KMeans and Mixture Models

K-means algorithm
Sec. 9.1 of Bishop PRML
http://research.microsoft.com/en-us/um/people/cmbishop/prml/

Gaussian mixture models and the EM algorithm
Sec. 9.2 of Bishop PRML
http://research.microsoft.com/en-us/um/people/cmbishop/prml/

Example stand-alone code (in Matlab and Python)
https://github.com/michaelchughes/compare-matlab-numpy

Connections between mixture models and EM
Sec. 9.3 of Bishop PRML

Bernoulli mixture models and the EM algorithm
Sec. 9.3.3 of Bishop PRML

## Markovian sequence models (HMMs, etc)

Markov chains review
http://cs.brown.edu/courses/csci1420/recitations/MarkovChains.pdf

Dynamic programming review

http://cs.brown.edu/courses/csci1420/recitations/dynamicProgrammingWithStringKernels.pdf

HMMs
See Chapter in Bishop

**Comment by hzhz2020** [comment]
Are these two links no longer valid?
http://cs.brown.edu/courses/csci1420/recitations/MarkovChains.pdf
http://cs.brown.edu/courses/csci1420/recitations/dynamicProgrammingWithStringKernels.pdf

**Comment by michaelchughes** [comment]
Yeah unfortunately I don't have access to those URLS. Will try to dig up those resources from my old laptops....
